script.js

```diff
@@ -1,4 +1,98 @@
+//Base
+// Requirements
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+var gameDiceRoll = "Game Dice Roll";
+var diceSelect = "Dice Selection";
+var gameMode = gameDiceRoll;
+var winner = "Determine the Winner";
+
+var currPlayer = 1;
+
+var diceRolled = [];
+var player1Results = "";
+var player2Results = "";
+
+var rollDice = function () {
+  var diceNum = Math.floor(Math.random() * 6 + 1);
+  return diceNum;
+};
+
+var arrayToNum = function (num1, num2) {
+  return Number(String(num1) + String(num2));
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameMode == gameDiceRoll) {
+    gameMode = diceSelect;
+    diceRolled = [rollDice(), rollDice()];
+    console.log(diceRolled);
+    return (
+      "Welcome Player " +
+      currPlayer +
+      ". <br> You rolled " +
+      diceRolled[0] +
+      " for Dice 1 and " +
+      diceRolled[1] +
+      " for Dice 2. <br> Choose the order of the dice."
+    );
+  }
+
+  if (gameMode == diceSelect) {
+    if (currPlayer == 1) {
+      if (input == 1) {
+        player1Results = arrayToNum(diceRolled[0], diceRolled[1]);
+      } else {
+        player1Results = arrayToNum(diceRolled[1], diceRolled[0]);
+      }
```

Comment:
This is a good quick way of deciding how to concatenate the dice numbers. However, this also means that as long as user enters anything other than 1, the result will be concatenating Dice2Dice1. 
Mebbe can think of a way to restrict entry to only 1 or 2. 
If user enters anything else, tell them it is invalid and try again?

This applies to lines 61 to 64